dcc-portal-ui/app/scripts/software/js/software.js

```diff
@@ -42,10 +42,18 @@
       Page.setPage('entity');
       Page.setTitle('Software Downloads');
 
-      $scope.artifactId = 'icgc-storage-client';
-      jQuery.get('api/v1/ui/software/' + $scope.artifactId + '/versions', function(versions) {
+      $scope.icgcStorageClient = {
+    		  artifactId: 'icgc-storage-client'
```

Comment:
Indentation seems off here. Please use the same indentation as the other js files.